grc.fraudDetection_web/src/main/java/com/typedb/examples/fraud/db/TransactionDao.java

```diff
@@ -97,6 +100,21 @@ public Set<Transaction> getName(String name) {
     return transactions;
   }
 
+  public Set<Transaction> getLimitOffset(int limit, int offset){
+
+    var matchLimitOffset = LIMIT_OFFSET_TX_MATCH.formatted(offset, limit);
+
+        var getQueryStr =
+            "match " + TX_MATCH + CardholderDao.CARDHOLDER_MATCH + BankDao.BANK_MATCH + MerchantDao.MERCHANT_MATCH + matchLimitOffset;
+
+    var results = db.getAll(getQueryStr);
+
+    var transactions = results.stream().map(TransactionDao::fromResult).collect(Collectors.toSet());
+
+    return transactions;
+
+  }
+
```

Comment:
@SullivanDaly Let's try to reuse the getAll(boolean suspect) method. For example, update it to add methods params for limit and offset. If limit != null, add the limit constraint. If offset is null, add the offset constraint. Then have getLimitOffset(int limit, int offset) call getAll(false, limit, offset)